src/State.sol

```diff
@@ -108,12 +114,18 @@ abstract contract State is OwnableUpgradeable, ReentrancyGuardUpgradeable {
     }
 
     function getRealBorrowAssets() public view returns (uint256) {
-        return lendingConnector.getRealBorrowAssets();
+        return lendingConnector().getRealBorrowAssets();
     }
 
     function getRealCollateralAssets() public view returns (uint256) {
-        return lendingConnector.getRealCollateralAssets();
+        return lendingConnector().getRealCollateralAssets();
     }
+
+    function lendingConnector() public view returns (ILendingConnector) {
```

Comment:
letsRename lendingConnector -> currentLendingConnector and keep lendingConnector without _